sonar-cxx-plugin/src/main/java/org/sonar/plugins/cxx/compiler/CxxCompilerSensor.java

```diff
@@ -47,6 +47,7 @@
   public static final String REPORT_CHARSET_DEF = "sonar.cxx.compiler.charset";
   public static final String PARSER_KEY_DEF = "sonar.cxx.compiler.parser";
   public static final String DEFAULT_PARSER_DEF = CxxCompilerVcParser.KEY;
+  public static final String DEFAULT_CHARSET_DEF = "UTF-16";
```

Comment:
@Bertk i think that one is yours, i had UTF-8 but i change to UTF-16 for some reason i cannot remember anymore